src/web/server/jaia.py

```diff
@@ -11,6 +11,7 @@
 from time import sleep
 import datetime
 from math import *
+import task_mapper
```

Comment:
I am getting an error when I try to run this. 

`Traceback (most recent call last):
  File "./app.py", line 9, in <module>
    import jaia
  File "/home/ferrom/repos/jaiabot/src/web/server/jaia.py", line 14, in <module>
    import task_mapper
ModuleNotFoundError: No module named 'task_mapper'`